helpers/discordBot.test.js

```diff
@@ -0,0 +1,122 @@
+process.env.C0D3R_ACCESS_TOKEN = 'cod3r:test'
+process.env.C0D3R_URL = 'c0d3r.mock.url'
+
+const {
+  sendChannelMessage,
+  sendLessonChannelMessage,
+  sendDirectMessage,
+  sendSubmissionNotification,
+  IdType
+} = require('./discordBot')
+
+const messageResponse = { id: '123456' }
+
+global.fetch = jest.fn(() =>
+  Promise.resolve({
+    json: () => Promise.resolve(messageResponse)
+  })
+)
+
+const apiEndpoint = `${process.env.C0D3R_URL}/api`
+const messagesEndpoint = `${apiEndpoint}/messages`
+
+const headers = {
+  Authorization: `Bearer ${process.env.C0D3R_ACCESS_TOKEN}`,
+  'Content-Type': 'application/json'
+}
+
+const channelId = '1234'
+const message = 'hello'
+const embed = { description: 'an embed' }
+const lessonId = 1
+const notificationLessonId = 2
+const idType = IdType.C0D3
+const userName = 'test_user'
+const discordUserId = '9898'
+const challengeTitle = 'test challenge'
```

Comment:
Can you please make this into an object instead? I think it's neater than having separate variables for everything.

Reply:
Might not look the neatest but it does make all the code below it very easy to read IMO

Reply:
```javascript
const mockData = {
  channelId: '1234',
  message: 'hello',
  embed: { description: 'an embed' },
  lessonId: 1,
  notificationLessonId: 2,
  idType: IdType.C0D3,
  userName: 'test_user',
  discordUserId: '9898',
  challengeTitle: 'test challenge'
}
```
Is this fine? variable names will be longer where used but should be readable